src/cmd/package.go

```diff
@@ -68,13 +75,23 @@ func choosePackage(args []string) string {
 	return path
 }
 
+func cachePathClean(cachePath string) bool {
+	var isCleanPath = regexp.MustCompile(`^[a-zA-Z0-9\_\-\/\.\~]+$`).MatchString
```

Comment:
I wonder how this might need to be dealt with when we add Windows support from #426?  Seems like https://pkg.go.dev/path/filepath might be needed in that case.  Thoughts?

Reply:
Okay, looking a bit closer at this. That [filepath](https://pkg.go.dev/path/filepath) lib doesn't seem to do input validation. The goal of the regex was to guard against malicious input/prevent unexpected behavior when a user sets the flag to `**/**` or something weird. In the case of Windows, I think we can just add the backslash char to the regex. Thoughts @jeff-mccoy ?

Reply:
(backslash char and a colon that is)

Reply:
yeah I think that's sufficient, I wasn't certain anything existed in that library and waiting to error out until later on maybe isn't ideal.